frontend/public/components/chargeback.tsx

```diff
@@ -377,19 +376,6 @@ const ReportsPage_: React.SFC<ReportsPageProps> = props => {
     </div>;
   }
   return <div>
```

Comment:
We should remove the FLAGS check above and always return lines 373-376

Reply:
Done.